torch/distributed/optim/zero_redundancy_optimizer.py

```diff
@@ -171,22 +262,37 @@ class ZeroRedundancyOptimizer(Optimizer, _Joinable):
         >>> ddp(inputs).sum().backward()
         >>> opt.step()
 
-    .. warning:
+    .. warning::
```

Comment:
I made these uniform across the file to use two colons. I checked the render, and the warning still appears as desired.